### PR TITLE
Use Samsung devices for Android URP tests

### PR DESCRIPTION
**Pull request by tomzig16**
### Purpose of this PR
This PR changes that Samsung devices are used for testing. This will help with shortening the waiting queues (we have more samsungs compared to old hikeys) and will be more precise to what user is using (because it is an actual device, compared to a non commercial board).

This PR also adds GLES3 reference images to ShaderGraph tests, which were previously missing. 3 of them are failing: https://fogbugz.unity3d.com/f/cases/1354427/

Moreover, in this PR I also update test framework graphics package version, as I have updated that package with a small feature that lets you assign a script and change resolution. This was necessary as mobile by default uses highest resolution possible, so samsungs were using `2280x1080`. All projects now are setting default resolution for Android devices to `1920x1080` (same as before, same as other platforms)

---
### Testing status
Tested on Yamato:
Custom test for every URP job + ShaderGraph
ABV

---
### Comments to reviewers
None

### CODE REVIEW

**Review by kivistikliis — COMMENTED** [review]
Thanks for the PR, besides the one question above, looks good to me! On a similar note though, should we have some pipelines on the non-samsungs still? 

And note to self @Unity-Technologies/gfx-sdets to prepare a separate PR to bump the gfx-tf version on other test projects too

File: .yamato/urp_performance_boatattack-android-vulkan.yml [hunk 1]
```diff
@@ -24,22 +24,18 @@ URP_Performance_BoatAttack_Android_Vulkan_performance_playmode_runtime_il2cpp_Li
         UTR_VERSION: "current"
         TEST_FILTER: .*
     commands:
-      -  command: '%ANDROID_SDK_ROOT%\platform-tools\adb.exe connect %BOKKEN_DEVICE_IP%'
```
[the remark this answers is not in the document]
Comment (tomzig16) [reply .yamato/urp_performance_boatattack-android-vulkan.yml:27]: Yes, this command in particular is not needed for Samsung devices as they are connected to Bokken agents via USB, while old hikeys are connected via wireless connection.

**Comment by tomzig16** [comment]
> Thanks for the PR, besides the one question above, looks good to me! On a similar note though, should we have some pipelines on the non-samsungs still?
> 
> And note to self @Unity-Technologies/gfx-sdets to prepare a separate PR to bump the gfx-tf version on other test projects too

@kivistikliis I am not sure about that. From what I know we will have hikey devices removed from the bokken, or maybe have 1-2 devices for some older tests in the future. I might have double check on the numbers, but pretty sure we want to move completely from hikeys to actual devices due to easier debugging and more close to end user environment.

File: .yamato/urp_performance_boatattack-android-vulkan.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (kivistikliis) [reply .yamato/urp_performance_boatattack-android-vulkan.yml:27]: Should all the performance projects be moved over to Samsungs as well in this case (I am unsure of the consequences of this, maybe we can get someone more on the Perf side to give an opinion on this)? Or we should add samsungs and hikeys as different platforms, such that the Perf jobs can keep using these commands?

**Review by theo-pnv — APPROVED** [review]

**Review by kivistikliis — APPROVED** [review]

**Comment by theo-pnv** [comment]
Hi, I am doing a little bit of cleaning regarding the SDET backports.
Considering there are some changes inside the test project and reference images, I think it is safer if your team handles this backport @tomzig16. Feel free to add the SDET team as reviewers when ready. Thanks!

**Pull request merged**